plugins/catalog-react/src/components/UserListPicker/useAllEntitiesCount.ts

```diff
@@ -30,13 +30,13 @@ export function useAllEntitiesCount() {
   const request = useMemo(() => {
     const { user, ...allFilters } = filters;
     const compacted = compact(Object.values(allFilters));
-    const filter = reduceCatalogFilters(compacted);
+    const catalogFilters = reduceCatalogFilters(compacted);
     const newRequest: QueryEntitiesInitialRequest = {
-      filter,
+      ...catalogFilters,
       limit: 0,
     };
 
-    if (Object.keys(filter).length === 0) {
+    if (Object.keys(catalogFilters).length === 0) {
```

Comment:
this should be

```suggestion
    if (Object.keys(catalogFilters.filter).length === 0) {
```

otherwise it is going to be always false